src/lib/scripts.js

```diff
@@ -89,8 +89,24 @@ const getScriptFieldValue = (contact, texter, fieldName) => {
   } else if (TOP_LEVEL_UPLOAD_FIELDS.indexOf(fieldName) !== -1) {
     result = contact[fieldName];
   } else {
-    const customFieldNames = JSON.parse(contact.customFields);
-    result = customFieldNames[fieldName];
+    // first try to match on texter extra fields
```

Comment:
It would be better to check here to see if the env var exists, but I'm not sure how to hook `getConfig` with `organization` here.